alacritty/src/config/bindings.rs

```diff
@@ -452,8 +452,9 @@ pub fn default_key_bindings() -> Vec<KeyBinding> {
             F20,    mods, ~TermMode::VI; Action::Esc(format!("\x1b[34;{}~", modifiers_code));
         ));
 
-        // We're adding the following bindings with `Shift` manually above, so skipping them here
-        // modifiers_code != Shift
+        // We're adding the following bindings with `Shift` manually above, so skipping them here.
+
+        // modifiers_code != Shift.
```

Comment:
Seems comment is broken-ish?

Reply:
I split it intentionally.

Reply:
I think it could be improved. Without a direct reference to some code, the comment kinda loses its reference point making it a bit unclear what specifically it refers to.

Reply:
We could also define shift etc as local constants, making the shift documentation redundant.